bitnami/fluentd/templates/aggregator-statefulset.yaml

```diff
@@ -0,0 +1,107 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ include "fluentd.fullname" . }}
+  labels:
+    {{- include "fluentd.labels" . | nindent 4 }}
+    app.kubernetes.io/component: aggregator
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "fluentd.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+      app.kubernetes.io/component: aggregator
```

Comment:
Create a `"fluentd.matchLabels"` macro for this. You can reuse it in the service definitions too: `svc.spec.selector`. Check other charts that implement this.